sdk-actors/src/test/java/io/dapr/actors/TestUtils.java

```diff
@@ -54,4 +55,12 @@ public static void assertThrowsDaprException(
     Assertions.assertEquals(expectedErrorCode, daprException.getErrorCode());
     Assertions.assertEquals(expectedErrorMessage, daprException.getMessage());
   }
+
+  public static String getSidecarIpForHttpUrl(final String sidecarIp) {
+    String retSidecarIp = sidecarIp;
+    if(InetAddressValidator.getInstance().isValidInet6Address(sidecarIp)) {
+      retSidecarIp = "[" + sidecarIp + "]";
```

Comment:
@willtsai Can you add a comment in this method which points to the format expected for IP v6?

Reply:
@mukundansundar - done, let me know if it looks good.